conex/behaviors/synapses/learning.py

```diff
@@ -249,14 +215,14 @@ def initialize(self, synapse):
 
         # messy till I move trace to synapse.
         pre_tau = [
-            synapse.src.behavior[key_behavior]
-            for key_behavior in synapse.src.behavior
-            if isinstance(synapse.src.behavior[key_behavior], SpikeTrace)
+            synapse.behavior[key_behavior]
+            for key_behavior in synapse.behavior
```

Comment:
This specific learning method needs variable tau_s to perform its operation; Since the trace now resides on synapse accessing the src neuron group is wrong.